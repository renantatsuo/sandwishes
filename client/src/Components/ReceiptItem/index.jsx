import React from 'react'
import { convertFloatToMoney } from '../../helpers'

const ReceiptItem = ({ className, price, text }) => {
  return (
    <div className={`receipt__item ${className}`}>
      <div className="receipt__item__desc">{text}</div>
      <div className="receipt__item__price">
        R$ {convertFloatToMoney(price)}
      </div>
    </div>
  )
}

export default ReceiptItem
