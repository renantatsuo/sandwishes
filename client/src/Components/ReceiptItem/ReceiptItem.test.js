import React from 'react'
import { shallow } from 'enzyme'
import ReceiptItem from './index'

describe('Test Ingredient component', () => {
  it('Should render values', () => {
    const props = {
      text: 'Test',
      price: 1
    }
    const receiptItem = shallow(<ReceiptItem {...props} />)

    expect(receiptItem.find('.receipt__item__price').text()).toMatch('R$ 1,00')
    expect(receiptItem.find('.receipt__item__desc').text()).toMatch('Test')
    expect(receiptItem).toMatchSnapshot()
  })
})
