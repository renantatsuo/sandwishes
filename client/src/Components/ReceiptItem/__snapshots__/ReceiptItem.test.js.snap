// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Ingredient component Should render values 1`] = `ShallowWrapper {}`;
